pushjack/apns.py

```diff
@@ -436,6 +466,7 @@ class APNSMessage(object):
     """APNs message object that serializes to JSON."""
     def __init__(self,
                  message=None,
+                 max_payload_length=None,
```

Comment:
The problem here is that it would be problematic for the client to do the truncation if it doesn't want to truncate more characters than needed. The APNS server is actually limiting the binary length of the whole payload, not just the message. I tweaked my PR so that the logic is off by default, and moved the param to the end of the argument list. Thank you for pointing that out, it makes total sense.